tests/src/test/scala/org/http4s/EntityDecoderSpec.scala

```diff
@@ -15,6 +21,10 @@ import scala.concurrent.ExecutionContext
 
 class EntityDecoderSpec extends Http4sSpec with PendingUntilFixed {
   implicit val executionContext: ExecutionContext = trampoline
+  implicit val testContext: TestContext = TestContext()
+
+  implicit def entityDecoderEq[A: Eq]: Eq[EntityDecoder[IO, A]] =
```

Comment:
Maybe we should have an `eq` package in our testing package for things that are "eq by lack of a counterexample," like cats does.